meraki/theme.json

```diff
@@ -110,11 +115,26 @@
 	},
 	"styles": {
 		"blocks": {
+			"core/button": {
```

Comment:
Since this is a Blockbase theme we leverage a lot of CUSTOM button values so that they can be taken advantage of in other areas (such as forms and buttons that aren't button blocks) and to achieve better control of hover states etc.  Thus, rather than changing the `styles.block.core/button` attributes directly it's better to change `custom.button` values instead.  See [Quadrat](https://github.com/Automattic/themes/blob/trunk/quadrat/theme.json#L62) for an example of overriding those button styles.